### PR TITLE
Implement Config with Corresponding Unit Tests.

### PR DESCRIPTION
**Pull request by yolandaw2021**
### Summary
Implemented the initialization of configuration of the simulation, to be passed into models.

### Testing
Have the init tested in different edge cases.

### Notes
The test is currently not runnable, waiting for setup.py file to be initialized.

### CODE REVIEW

File: tests/core_tests/config_test.py [hunk 1]
```diff
@@ -1,13 +1,60 @@
...
+from src.core.config import Config
+from src.core.parameters import Parameters
+
 
 DEBUG = False
 
 class ConfigTestCases(unittest.TestCase):
-    pass
-    # TODO
+    '''This test tests whether '''
```
Comment (tmf97) [inline tests/core_tests/config_test.py:10]: whether what?

File: src/core/config.py [hunk 2]
```diff
@@ -1,3 +1,30 @@
...
+            "pos_y": 0.0,
+            "pos_z": 0.0,
+            "pos_ang": 0.0,
+            "velocity": 0.0,
+            "quad_rate": 0.0,
+        }
+
+        for key, value in initial_conditions.items():
+            if key in default_conditions.keys():
+                setattr(default_conditions, key, value)
```
Comment (tmf97) [inline src/core/config.py:29]: you don't need to use `setattr` here, you can just say `default_conditions[key] = value`

Comment (tmf97) [reply src/core/config.py:29]: `setattr` is kinda dangerous because you could be setting something that doesn't actually exist yet

**Review by tmf97 — CHANGES_REQUESTED** [review]

**Review by tmf97 — APPROVED** [review]
Sweeeeeeeeeeeeeeeeeeeeeeeeeet
